Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -424,8 +424,11 @@ static ContentValues SetupContentValues(CalendarEvent newEvent)
             eventValues.Put(CalendarContract.Events.InterfaceConsts.Dtend, newEvent.EndDate.HasValue ? newEvent.EndDate.Value.ToUnixTimeMilliseconds().ToString() : newEvent.StartDate.AddDays(1).ToUnixTimeMilliseconds().ToString());
             eventValues.Put(CalendarContract.Events.InterfaceConsts.EventTimezone, TimeZoneInfo.Local.Id);
             eventValues.Put(CalendarContract.Events.InterfaceConsts.EventEndTimezone, TimeZoneInfo.Local.Id);
-            eventValues.Put(CalendarContract.Events.InterfaceConsts.Deleted, 0);
-            eventValues.Put(CalendarContract.Events.InterfaceConsts.Rrule, newEvent.RecurrancePattern.ConvertRule());
+            if (newEvent.RecurrancePattern != null)
+            {
+                eventValues.Put(CalendarContract.Events.InterfaceConsts.Rrule, newEvent.RecurrancePattern.ConvertRule());
```

Comment:
In the case of updating an existing event, if newEvent doesn't have a recurrance pattern, do we need to clear the Rrule property (just in case the existing event has a recurrence pattern set)?

Reply:
Hmm good point, I will have to think about this one.

Reply:
Have altered recurrence rules so that you can now remove them from existing events.